models/meshmodel/core/policies/rego_policy_relationship.go

```diff
@@ -51,7 +56,10 @@ func (r *Rego) RegoPolicyHandler(regoQueryString string, designFile []byte) (int
 	}
 	regoEngine, err := rego.New(
 		rego.Query(regoQueryString),
-		rego.Load([]string{r.policyDir}, nil),
+		rego.Load([]string{r.policyDir}, func(abspath string, info fs.FileInfo, depth int) bool {
```

Comment:
Please add comments here and update Meshery Docs. 

Is this loading Rego policy from each individual model?